src/config.js

```diff
@@ -29,7 +29,7 @@ global.config = {
 
   info: {
     signController: true,
-    signText: 'Fully automated TooAngel bot: https://github.com/TooAngel/screeps',
+    signText: 'Fully automated TooAngel bot: http://tooangel.github.io/screeps/',
```

Comment:
on http://tooangel.github.io/screeps/ the last line are not well formated (maybe missing newlines)